package.json

```diff
@@ -42,14 +42,13 @@
   },
   "devDependencies": {
     "chai": "4.3.6",
-    "mocha": "10.0.0",
+    "import-fresh": "3.3.0",
+    "mocha": "6.2.3",
```

Comment:
is the newer version of mocha the responsible library for producing these warnings?